dll/shellext/zipfldr/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+
+set_cpp(WITH_RUNTIME WITH_EXCEPTIONS)
+if(NOT MSVC)
+    # HACK: this should be enabled globally!
+    add_compile_flags_language("-std=c++11" "CXX")
```

Comment:
sorry thomas ;)

Reply:
Isn't it? AFAIK rapps builds with this flag.

Reply:
This doesn't seem to indicate so: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=c%2B%2B11